app/src/main/contrib/components/Responsibilities/Coverage/DownloadCoverageContent.tsx

```diff
@@ -65,6 +76,19 @@ export class DownloadCoverageContentComponent extends RemoteContentComponent<Dow
         responsibilityStore.fetchOneTimeCoverageToken();
     }
 
+    onDownloadClicked() {
+        setTimeout(() => {
+            this.setState({
+                downloadButtonEnabled: false,
+            })
+        }, 50)
+        setTimeout(() => {
+            this.setState({
+                downloadButtonEnabled: true,
+            })
+        }, 5000)
```

Comment:
Maybe more readable as

```
setTimeout(() => this.setState({
    downloadButtonEnabled: true
}), 5000);
```

But that's personal preference. Equally the other statement. Also, it might be nice to pull the 5000ms delay into a named static const?

Reply:
Although see below about idea to parameterize delay.